webcompat/static/js/lib/models/issue.js

```diff
@@ -43,6 +43,7 @@ export const Issue = Backbone.Model.extend({
     var milestoneClass;
     if (response.milestone) {
       milestone = response.milestone.title;
+      milestoneClass = response.milestone.title;
     } else {
```

Comment:
I've noticed that milestone labels were not coloured on the issue page, because a css class was missing, this is a fix for that.

currently:
<img width="522" alt="Screen Shot 2021-11-18 at 5 53 12 PM" src="https://user-images.githubusercontent.com/1303908/142509688-31163a6f-dcd3-42e9-a64c-59ed9fb3a837.png">
will be:
<img width="562" alt="Screen Shot 2021-11-18 at 5 53 32 PM" src="https://user-images.githubusercontent.com/1303908/142509696-4628de28-2437-4542-86b4-4e3ca901b9c0.png">